src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java

```diff
@@ -57,10 +61,40 @@
     private WriteBatch batch;
     @Mock
     private StorageDao storageDao;
+    @Mock
+    private DocumentSnapshot documentSnapshot;
 
     @InjectMocks
     private PetDao petDao = new PetDaoImpl();
 
+    @BeforeEach
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [NCSS for this method is 23 (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)